index.html

```diff
@@ -2003,23 +2027,16 @@
             presentations</a>.
             </li>
             <li>
-              <a>Queue a task</a> to <a>fire</a> an event named
-              <code>connectionavailable</code> at all
-              <a>PresentationConnectionList</a>.
+              <a>Queue a task</a> to <a>fire</a> a <a>trusted event</a> with
+              the name <code>connectionavailable</code>, that uses the
+              <a>PresentationConnectionAvailableEvent</a> interface, with the
+              <a for="PresentationConnectionAvailableEvent">connection</a>
+              attribute initialized to <var>S</var>, at the
+              <a>PresentationConnectionList</a> instance associated with the
+              <a>PresentationReceiver</a> object.
```

Comment:
Did you want to specify the bubble/cancel/default action behavior for this event, like the other trusted events?